NineChronicles.DataProvider.Executable/Program.cs

```diff
@@ -131,9 +131,25 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                 {
                     services.AddDbContextFactory<NineChroniclesContext>(options =>
                     {
+                        string[] input = new string[1];
+                        Console.Write("Please enter your MySql connection string (Just press \"Enter\" to use SqlLite): ");
```

Comment:
I guess that initeractive configuration can help human operator, but it matters other automated running environment because it will needs standard intput even in environments where it is difficult to use(e.g.,  CI/CD)

so how about using configuration as `IConfiguration`  like `headlessConfig`?